src/proto/message_translation/power_frame_msg.hpp

```diff
@@ -0,0 +1,218 @@
+#pragma once
+
+#include <pb_decode.h>
+#include <pb_encode.h>
+
+#include <cstdint>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#ifdef PLATFORMIO_BUILD
+#include <proto/power_frame_msg.nanopb.h>
+#else
```

Comment:
this really helps readability 
```suggestion
#else // PLATFORMIO_BUILD
```